ledger/puzzle/src/solution/serialize.rs

```diff
@@ -19,8 +19,7 @@ impl<N: Network> Serialize for Solution<N> {
     fn serialize<S: Serializer>(&self, serializer: S) -> Result<S::Ok, S::Error> {
         match serializer.is_human_readable() {
             true => {
-                let mut partial_solution = serializer.serialize_struct("Solution", 4)?;
-                partial_solution.serialize_field("solution_id", &self.solution_id)?;
```

Comment:
We actually want to store the `solution_id` in the Solution. Mainly so we have a unique reference to solutions without having to compute it.

Reply:
Yes, revert these changes